macros/cross_db_utils.sql

```diff
@@ -33,10 +37,6 @@
   {% do return(is_numeric) %}
 {%- endmacro -%}
 
-{%- macro athena__is_numeric_dtype(dtype) -%}
```

Comment:
Why is this removed?

Reply:
This seems to be sync issue; correcting the same.